sources/api/netdog/src/cli/write_network_status.rs

```diff
@@ -24,12 +22,11 @@ pub(crate) fn run() -> Result<()> {
         .primary_address()
         .context(error::PrimaryInterfaceAddressSnafu {})?;
     write_current_ip(primary_ip)?;
-
-    // Write out resolv.conf
-    write_resolv_conf(&primary_link_status)?;
-
     write_primary_interface_sysctl(primary_interface)?;
 
+    // Symlink resolv.conf to a common path
+    force_symlink(REAL_RESOLV_CONF, NETDOG_RESOLV_CONF)?;
```

Comment:
I lost the thread here for a few minutes while comparing this to the wicked helper path.

Maybe a comment:
```suggestion
    // Symlink resolv.conf to a common path.
    // We don't ever write a resolv.conf when using networkd with resolved;
    // instead we write a drop-in configuration for resolved when the API
    // settings change.
    force_symlink(REAL_RESOLV_CONF, NETDOG_RESOLV_CONF)?;
```